cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml

```diff
@@ -219,7 +219,7 @@ phases:
               
               # Download cookbook
               mkdir -p /etc/chef && sudo chown -R root:root /etc/chef
-              curl --retry 3 -L -o /etc/chef/aws-parallelcluster-cookbook.tgz "$COOKBOOK_URL"
+              curl --retry 10 --retry-all-errors -L -o /etc/chef/aws-parallelcluster-cookbook.tgz "$COOKBOOK_URL"
```

Comment:
How many retires did it need for the test to pass in china? I think 17 mins is too high we add this 17 mins overall
Lets keep a smaller as the time to actually connect to the URL and then fail is still not counted. So far as you mentioned it never retired and failed within for example 2 mins.
So with every 1 + 2 + 4 + 8 + 16 + 32 + 64 + 128 + 256 + 512 = 1023 seconds = 17.05 minutes we would have to add the 2 mins in-between to get the actual increase in time